transport-native-epoll/src/main/java/io/netty/channel/epoll/NativeDatagramPacketArray.java

```diff
@@ -117,6 +117,8 @@ void release() {
         private int count;
 
         private final byte[] addr = new byte[16];
+        private final byte[] ipv4Bytes = new byte[4];
```

Comment:
Maybe add a comment to make it clear this is just a temp array to construct ipv4 `InetAddress`? kind of different to the other "value" fields

Reply:
yep let me do this...

Reply:
let me even make it part of the outer-class to save some memory.

Reply:
@njhill done